### ISSUE TITLE
Add dynamic label-name to Logo for accessibility

### ISSUE THREAD

**Issue by jiromaykin** [open-formulieren/open-forms#2428]
### Thema / Theme

Frontend

### Omschrijving / Description

A dynamic title on the Logo, that can be read by screenreaders. is needed. On the aria-label and on the title. Right now the image just reads statically as "terug naar website".
**1.** Goal is to change it to: "terug naar website [**organisatienaam**]"
Where "organisatienaam" should be the dynamic name of the municipality and could be a new config field.
So this issue is a request to add that dynamic name + it should be part of translations (_Back to website organisationname / Terug naar website organisationname_), possibly by use of Blocktrans.

**2.** Additionally the current code is not up to accessibility standards, as only IMG tags and SVG's are allowed to have an "alt".
So the **alt** "_Logo gemeente_" inside the anchor will not be read by screenreaders and should be removed.

**3.** This new config field should be added to the organisation name to both the **aria-label** and the **title** attribute. Both with a translatable "Back to website **organisationname**".

### Added value / Toegevoegde waarde

A dynamic title will give an accessible meaning to the organization logo + correct syntax also makes the image-role more accessible for screen-readers.

### Aanvullende opmerkingen / Additional context

![header-alt](https://user-images.githubusercontent.com/118177951/205311991-e0b1ec92-62ac-4668-8e93-9b2538573d41.png)

**Comment by jiromaykin** [comment]
Currently I do not see a configuration field in the dashboard yet (should be near Gemeentelogo on admin/config/globalconfiguration/), so reopening this issue.
